demo-kotlin/app/src/main/java/com/example/placesdemo/PlaceIsOpenActivity.kt

```diff
@@ -129,13 +129,10 @@ class PlaceIsOpenActivity : AppCompatActivity() {
         val placeTask: Task<IsOpenResponse> = placesClient.isOpen(request)
         placeTask.addOnSuccessListener { response ->
             binding.textViewResponse.text =
-                "Is place open? " + response.isOpen + "\nExtra place details: \n" + place?.let {
-                    stringify(
-                        it
-                    )
-                }
+                "Is place open? " + response.isOpen + "\nExtra place details: \n" + stringify(place)
```

Comment:
nit: what about using a StringBuilder or String interpolation? It is slightly more effective from a compiler perspective, since the operator + creates new strings.

Reply:
done